Editing the Cloud Environment Variables doc (#1791)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables.md

**Setting environment variables at the project and environment level**

> To set environment variables at the project and environment level, click **Deploy** in the top left, then select **Environments**.  Click Environments Variables to add and update your environment variables. 

Reviewer: ```suggestion
To set environment variables at the project and environment level, click **Deploy** in the top left, then select **Environments**.  Click **Environments Variables** to add and update your environment variables. 
```

> To set environment variables at the project and environment level, head over to the Environments section under Deploy in your top navigation. You'll see a view to list all your environments and a view to set and update environment variables. Make sure you're in the environment variable view.

Reviewer: ```suggestion
To set environment variables at the project and environment level, click **Deploy** in the top left, then select **Environments**.  Click Environments Variables to add and update your environment variables. 
```

**Overriding environment variables at the personal level**

> Developers may also want to set a personal value override for an environment variable while developing in the dbt IDE. By default, dbt Cloud will look to use environment variable values set in the project's development environment. To see and override these values, developers should head to the Settings page. Under Your Profile section, developers can select their project under Credentials and scroll to the Environment Variables section. 

Reviewer: ```suggestion
You can also set a personal value override for an environment variable when you develop in the dbt integrated developer environment (IDE). By default, dbt Cloud uses environment variable values set in the project's development environment. To see and override these values, click the gear icon in the top right. Under "Your Profile," click **Credentials** and select your project. Click **Edit** and make any changes in "Environment Variables." 
```
